subiquity/models/tests/test_filesystem.py

```diff
@@ -105,6 +106,12 @@ def test_errors(self):
                 self.assertEqual(expected_error, actual_error)
 
 
+class TestRoundRaidSize(unittest.TestCase):
+
+    def test_lp1816777(self):
+        self.assertLessEqual(round_raid_size(500107862016), 499972571136)
```

Comment:
4.6 TiB drive has 130 MB overhead.  OK.  It would be interesting to see the mdadm --examine superblock dump to match up where the space went (bitmap, alignment, etc).

Reply:
Sadly none of our bug reporters we quite as thorough as to include that :)